### DISCUSSION TITLE
Where do I set up notifications for download completion?

### DISCUSSION THREAD

**Discussion by Tanggue** [aclap-dev/video-downloadhelper#1235] [Help and Q&A]
### Your question

Before the interface was updated, you could set a notification pop-up when the download was complete, but in the new version, the notification settings interface is not visible. Where can I set the notification settings?

### Addon & CoApp version and details

Version 9.1.0.42 (stable)
target mozilla

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Open the VDH settings.  At the bottom of that menu there is a `More settings` entry.  Click that.  That opens yet another window of settings.  Scroll it down.  You will see 2 settings controlled by check boxes that deal with notifications.

**Reply by Tanggue** [reply]
I didn't know you could scroll. Thanks for this, it worked.

**Reply by GrampaWildWilly** [reply]
And you can use the VDH menu -- including if you use the sidebar -- mostly from the keyboard.  The mouse is just barely needed at all.  One of the things I like most about this new interface.
